### PR TITLE
Move the fcn refs visual feature into the view instead of adding more keys

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: libr/core/visual.c [hunk 1]
```diff
@@ -1036,20 +1036,20 @@ R_API int r_core_visual_refs(RCore *core, bool xref) {
 	RList *xrefs = NULL;
 	RAnalRef *refi;
 	RListIter *iter;
-	RAnalFunction *fun;
 	int skip = 0;
 	int idx = 0;
 	char cstr[32];
 	ut64 addr = core->offset;
 	int printMode = 0;
+	int xrefsMode = 0;
```
Comment (ret2libc) [inline libr/core/visual.c]: why not a bool?

File: libr/core/visual.c [hunk 2]
```diff
@@ -1036,20 +1036,20 @@ R_API int r_core_visual_refs(RCore *core, bool xref) {
...
 	if (core->print->cur_enabled) {
 		addr += core->print->cur;
 	}
 
 repeat:
-	fun = r_anal_get_fcn_at (core->anal, addr, R_ANAL_FCN_TYPE_NULL);
-	if (fun) {
+	if (xrefsMode % 2) {
+		RAnalFunction *fun = r_anal_get_fcn_at (core->anal, addr, R_ANAL_FCN_TYPE_NULL);
 		if (xref) {
```
Comment (ret2libc) [inline libr/core/visual.c]: you are not checking anymore if `fun` is not null.

File: libr/core/visual.c [hunk 3]
```diff
@@ -1188,10 +1193,15 @@ R_API int r_core_visual_refs(RCore *core, bool xref) {
 		" pP  - rotate between various print modes\n"
 		" :   - run r2 command\n"
 		" ?   - show this help message\n"
+		" TAB - toggle between address and function references\n"
+		" xX  - switch to refs or xrefs\n"
 		" \\n  - seek to this xref");
```
Comment (Maijin) [inline libr/core/visual.c]: Can you add `q` ?

**Comment by radare** [comment]
Try again i did some more improvements late at night

**Pull request merged**